openapi.yaml

```diff
@@ -330,15 +330,15 @@ components:
         quotaMaxBytes:
           type: integer
           format: int64
-          description: "Maximum storage space used by the node"
+          description: "Maximum storage space (in bytes) used by the node on the hard drive."
         quotaUsedBytes:
           type: integer
           format: int64
-          description: "Amount of storage space currently in use"
+          description: "Amount of storage space (in bytes) currently in use by availabilities for existing files."
         quotaReservedBytes:
           type: integer
           format: int64
-          description: "Amount of storage space reserved"
+          description: "Amount of storage allocated (in bytes) for the node by availabilities for future use when storage requests are fulfilled. This does not include the storage currently in use."
```

Comment:
```suggestion
          description: "Amount of storage reserved (in bytes) in the Codex's local repository for future use when storage requests will be picked up and hosted by the node using node's availabilities. This does not include the storage currently in use."
```

Added the "Codex's local repository" and bit rephrased. If you don't like the rewording, then just please include the "Codex's local repository".